velPaint

```diff
@@ -4,6 +4,9 @@
 # International License](http://creativecommons.org/licenses/by/4.0/).
 # Based on a work at https://github.com/MarkWheadon/velocity-painting.
 
+use strict;
+use warnings;
+use 5.10.1;
```

Comment:
If run on a pre-5.10 version of the interpreter, this will error with a useful message instead of failing later on the %+ variable.